bin/test

```diff
@@ -1,12 +1,17 @@
 #!/bin/sh
 
-# Set test variables
-export DATABASE_URL=mysql2://root:secret@mysql/test
+# We use TEST_DATABASE_URL to make sure we don't accidentaly overwrite dev/prod db
+unset DATABASE_URL; export DATABASE_URL
```

Comment:
ok. So there has to be set either the TEST_DATABASE_URL or the database.yml. Thats' o.k. for me.